src/components/viewSections/LicenseInternalContacts.js

```diff
@@ -47,7 +47,7 @@ export default class LicenseInternalContacts extends React.Component {
         displayWhenClosed={this.renderBadge()}
         displayWhenOpen={this.renderBadge()}
         id={id}
-        label={<FormattedMessage id="ui-agreements.agreements.internalContacts" />}
+        label={<FormattedMessage id="ui-licenses.emptyAccordion.internalContacts" />}
```

Comment:
Missed this yesterday....this is actually the label of the accordion..
so id needs to be `ui-licenses.section.internalContacts`

Reply:
Raised a PR [here](https://github.com/folio-org/ui-licenses/pull/319)